cabra/src/pages/components/SubtasksListEdit.tsx

```diff
@@ -87,31 +74,35 @@ export default function SubtasksListEdit({ parentId }: Props) {
 
   return (
     <div>
-      <form onSubmit={handleSubmitSubtask}>
-        <div tw="w-full flex gap-4 mb-4">
-          <SubtaskInput
-            id="subtask"
-            maxLength={80}
-            onChange={(event) => setVal(event.target.value)}
-            placeholder="Subtask name"
-            required
-            type="text"
-            value={val}
+      <Container>
+        {subtasks.map((relatedTask) => (
+          <MiniTaskDelete
+            key={relatedTask.relationId}
+            title={relatedTask.task.title}
+            onClickDeleteTask={removeTask.mutateAsync}
+            onClickRemoveRelation={handleClickDeleteRelation}
+            taskId={relatedTask.task.id}
+            relationType={relationType}
           />
-          <AddSubtaskButton type="submit">
-            <PlusIcon width={24} height={24} /> Add
-          </AddSubtaskButton>
-        </div>
-        <Container>
-          {subtasks.map((relatedTask) => (
-            <MiniTaskDelete
-              key={relatedTask.relationId}
-              title={relatedTask.task.title}
-              onClickDelete={removeTask.mutateAsync}
-              taskId={relatedTask.task.id}
+        ))}
+      </Container>
+      <form onSubmit={handleSubmitSubtask}>
+        <div tw="rounded-xl w-full text-blue-800  p-1.5">
+          <p tw="flex items-center">
+            <input
+              css={[tw`form-input`, fieldStyles]}
+              id="subtask"
+              type="text"
+              value={val}
+              maxLength={80}
+              required
+              onChange={(event) => setVal(event.target.value)}
             />
-          ))}
-        </Container>
+          </p>
+          <p tw="flex items-center">
+            <AddDependenceButton type="submit">+add</AddDependenceButton>
```

Comment:
Use styles for Submit button from `RelationListEdit`. It'll be more consistent and working for sure.